config/ROOT.scons

```diff
@@ -22,13 +22,28 @@ rootdict = Builder(generator = generate_action,
                    src_suffix = '.hh')
 ####
 
+import subprocess
+
+desired = ['python','minuit2']
```

Comment:
@mastbaum @nbarros This seemed to be the most robust way to check that ROOT was built with support for various things, and I added an explicit check for python support for good measure. Thoughts/suggestions?